src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -239,26 +240,32 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
   })
 
   useEffect(() => {
-    if (!(txId && tx)) return
+    if (!txId) return
 
-    if (tx?.status === TxStatus.Pending) {
-      setStatus(tx.status)
+    // Avoids this hook's mutate fn running too many times
+    if (status === TxStatus.Confirmed || status === TxStatus.Failed) return
```

Comment:
Can we add another comment explaining the why of this?

Reply:
Also, do we want to poll on pending Tx *only* for the RUNE case? e.g BTC or BCH can be quite long to 1 confirmation, and this means we would effectively be polling for nothing while waiting for first confirmation, possibly exhausting the timeout because of this change

Reply:
only rune transactions will immediately start polling, everything else will wait for the transaction to be confirmed on chain before polling